web-ui/src/main/resources/catalog/views/default/templates/searchForm.html

```diff
@@ -30,53 +33,39 @@
                     data-typeahead-wait-ms="600">
 
           <div class="input-group-btn">
-
-            <button type="button"
-                    data-ng-click="triggerSearch()"
-                    title="{{'search' | translate}}"
-                    class="btn btn-primary btn-lg">
-              <i class="fa fa-fw fa-search"></i>
-              <span class="sr-only" data-translate="">search</span>
-            </button>
-
             <button type="button"
                     data-ng-click="resetSearch(searchObj.defaultParams);"
                     title="{{'ClearTitle' | translate}}"
                     data-ng-if="::!isFilterTagsDisplayedInSearch"
```

Comment:
Maybe we should always display reset action here even if the active filter widget is not on. For example, if you do a full text search and no facet selected, there is no action to reset the search. The active filter is only visible if one facet is selected. What do you think @MichelGabriel ?

Reply:
For me it makes sense to always show the `reset` button, exactly for the reasons you mention above